content/doc/book/installing/_docker.adoc

```diff
@@ -13,264 +13,129 @@ _run-jenkins-in-docker.adoc).
-containers controlled by this Docker container's Docker daemon to mount data
-from Jenkins.
-<10> ( _Optional_ ) Exposes the Docker daemon port on the host machine. This is
-useful for executing `docker` commands on the host machine to control this
-inner Docker daemon.
-<11> The `docker:dind` image itself. This image can be downloaded before running
-by using the command: `docker image pull docker:dind`.
-+
-*Note:* If copying and pasting the command snippet above does not work, try
-copying and pasting this annotation-free version here:
+FROM jenkins/jenkins:2.249.1-lts
+USER root
+RUN apt-get update && apt-get install -y \
+       apt-transport-https \
+       ca-certificates \
+       curl \
+       gnupg2 \
+       software-properties-common
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+RUN apt-key fingerprint 0EBFCD88
+RUN add-apt-repository \
+       "deb [arch=amd64] https://download.docker.com/linux/debian \
+       $(lsb_release -cs) \
+       stable"
+RUN apt-get update && apt-get install -y docker-ce-cli
+USER jenkins
+RUN /usr/local/bin/install-plugins.sh blueocean:1.24.0
+
+ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/jenkins.sh"]
+----
+.. Build a new docker image from this Dockerfile and assign the image meaningful name, e.g. "myjenkins:1.0":
 +
-[source,bash]
+[source]
 ----
-docker container run --name jenkins-docker --rm --detach \
-  --privileged --network jenkins --network-alias docker \
-  --env DOCKER_TLS_CERTDIR=/certs \
-  --volume jenkins-docker-certs:/certs/client \
-  --volume jenkins-data:/var/jenkins_home \
-  --publish 2376:2376 docker:dind
+docker build -t "myjenkins:1.0" .
 ----
-. Download the `jenkinsci/blueocean` image and run it as a container in Docker
-  using the following
-  link:https://docs.docker.com/engine/reference/commandline/container_run/[`docker container run`]
+Keep in mind that described above process will automatically download official Jenkins Docker image 
+if this hasn't been done before.
+
+. Run your own `myjenkins:1.0` image as a container in Docker using the
+  following
+  link:https://docs.docker.com/engine/reference/run/[`docker run`]
   command:
 +
-[source,bash]
+[source]
 ----
-docker container run \
-  --name jenkins-blueocean \# <1>
-  --rm \# <2>
-  --detach \# <3>
-  --network jenkins \# <4>
-  --env DOCKER_HOST=tcp://docker:2376 \# <5>
-  --env DOCKER_CERT_PATH=/certs/client \
-  --env DOCKER_TLS_VERIFY=1 \
-  --publish 8080:8080 \# <6>
-  --publish 50000:50000 \# <7>
-  --volume jenkins-data:/var/jenkins_home \# <8>
-  --volume jenkins-docker-certs:/certs/client:ro \# <9>
-  jenkinsci/blueocean# <10>
+docker run --rm -u root --name jenkins-dkr-blueocean \
+  --volume jenkins-data:/var/jenkins_home \ # <1>
+  --volume "$HOME":/home \ # <2>
+  --volume /var/run/docker.sock:/var/run/docker.sock \ # <3>
+  --publish 8080:8080 \ # <4>
+  --publish 50000:50000 \# <5>
+  myjenkins:1.0
```

Comment:
When I tried to copy this from the generated page and paste it into my terminal window, the ` #` at the end of the lines made it fail.  I think cut and paste will work if you change it to:
```suggestion
  --volume jenkins-data:/var/jenkins_home \# <1>
  --volume "$HOME":/home \# <2>
  --volume /var/run/docker.sock:/var/run/docker.sock \# <3>
  --publish 8080:8080 \# <4>
  --publish 50000:50000 \# <5>
  myjenkins:1.0
```